app/code/Magento/Payment/Gateway/Data/AddressAdapterInterface.php

```diff
@@ -40,6 +40,20 @@ public function getStreetLine1();
      */
     public function getStreetLine2();
 
+    /**
+     * Get street line 3
+     *
+     * @return string
+     */
+    public function getStreetLine3();
```

Comment:
This interface is marked as api, we can't introduce new methods to existing api.
If you have specific requirements for your project you can use the extension attribute.

Reply:
@Stepa4man 
I understand, but note that what I try to get is information that already exists on the platform, they are not new fields, they are just fields that already exist on the platform, but they are not mapped in the API.
We are talking about lines 3 and 4 of the vat_id (address attribute) of the street.

Reply:
You can get this information in your customization module. We can't change api for Magento as a product because one of the projects needs this data.